tasks/skylab/LoomUtils.wdl

```diff
@@ -196,3 +196,100 @@ task AggregateSmartSeq2Loom {
       maxRetries: 1
     }
 }
+
+
+task SingleNucleiSmartSeq2LoomOutput {
```

Comment:
Could we mention in a comment here why we needed an additional task to handle the single nuclei loom output vs the single cell?

Reply:
Updated the description of the task.